packages/@lwc/integration-karma/scripts/karma-plugins/transform-framework.js

```diff
@@ -11,9 +11,8 @@
  */
 'use strict';
 
-const path = require('path');
-const { rollup } = require('rollup');
-const replace = require('@rollup/plugin-replace');
+const MagicString = require('magic-string');
+const { init, parse } = require('es-module-lexer');
```

Comment:
I switched from using Rollup to using `magic-string` and `es-module-lexer` to transform ESM into IIFE.

This new method should be a lot faster, since we really don't need all of Rollup just to transform our framework files (e.g. `engine-dom/dist/index.js`). These files don't have any imports, and just need some light editing to replace some strings and convert ESM to IIFE.